openmp/libomptarget/test/offloading/fortran/target-region-implicit-array.f90

```diff
@@ -14,14 +14,14 @@ program main
     integer :: i = 1
     integer :: j = 11
 
-  !$omp target map(tofrom:x, i, j)
-     do while (i <= j)
+  !$omp target
+     do i = 1, j
         x(i) = i;
         i = i + 1
```

Comment:
I'm not sure if this will compile at the moment because of the `i = i +1`, I think it will complain about a redefine I might be incorrect though! And I am not sure the check value would be the same with the line either. I could be wrong however!

I'd suggest just changing it to something like the below and drop the external integers (the `i` will likely get implicitly captured without implicit none in any case):

  !$omp target
     do i = 1, 10
        x(i) = i;
    end do
  !$omp end target

Reply:
Sorry, I changed it in my local test and forgot to update the actual test. Fixed now!

Reply:
Thank you Akash! And I've done that on more than one occasion now unfortunately (you're fixing one that sadly landed upstream in this PR)!